internal/auth0/client/global_resource.go

```diff
@@ -69,7 +70,7 @@ func readGlobalClientID(ctx context.Context, d *schema.ResourceData, m interface
 	return nil
 }
 
-func deleteGlobalClient(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+func deleteGlobalClient(_ context.Context, d *schema.ResourceData, _ interface{}) diag.Diagnostics {
```

Comment:
Should the context and interface params be removed if they're not used? If so, maybe in another PR.

Reply:
This might have been a bit pre-mature, however the long term plan for the Go SDK is to add context support on the function calls, and seeing that all the signatures for the terraform CRUD funcs use context, it made sense to keep this consistent as well.